recipe/meta.yaml

```diff
@@ -17,8 +17,14 @@ source:
     # Guard deprecated nanohttp cleanup for libxml2 2.14 where HTTP may be disabled
     - 0004-guard-nanohttp-cleanup.patch
 
+{% set build = 17 %}
```

Comment:
Let's move this to be right under the version at the top of the file.  Easier to find.